.gitignore

```diff
@@ -42,3 +42,5 @@ nb-configuration.xml
 
 ##Ignore JRebel project configuration
 rebel.xml
+
+test.csv
```

Comment:
Please remove this change to `.gitignore`.  The bug that resulted in a `test.csv` sitting around (after tests) was resolved in #2497.  If you update this PR to latest `master` (via a rebase) you no longer should see the `test.csv`.